build/lib/release.sh

```diff
@@ -28,7 +28,7 @@ readonly RELEASE_STAGE="${LOCAL_OUTPUT_ROOT}/release-stage"
 readonly RELEASE_TARS="${LOCAL_OUTPUT_ROOT}/release-tars"
 readonly RELEASE_IMAGES="${LOCAL_OUTPUT_ROOT}/release-images"
 
-KUBE_BUILD_CONFORMANCE=${KUBE_BUILD_CONFORMANCE:-y}
+KUBE_BUILD_CONFORMANCE=${KUBE_BUILD_CONFORMANCE:-n}
```

Comment:
I feel like we should invert so this defaults to Y and quick targets narrow down explicitly. But this is the shorter diff to accomplish the same thing, let's see what the effects are and I can file follow up if I care